examples/native/components/CSFExample/BooleanExampleButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent } from 'react';
+import { TouchableOpacity, StyleSheet, Text } from 'react-native';
+
+export interface Props {
+  isEnabled: boolean;
+}
+
+const styles = StyleSheet.create({
+  button: { paddingHorizontal: 16, paddingVertical: 8 },
+});
+
+export const BooleanExampleButton: FunctionComponent<Props> = ({ isEnabled }: Props) => (
```

Comment:
See the PR description - how do we want to structure the examples?

Reply:
This is something I'm also not sure about, previously there was one component that had examples of each control. However I do also like the idea of having stories for each one. Interested to know what your opinion is too.

Reply:
Tough one 🤔 . Trying to list some arguments for both alternatives:

**One component with multiple controls**
+ Faster to check all control functionality (no need to jump between stories)
+ Less code

**One story per control**
+ Explicit about what code is necessary to use this control
+ Could be clearer to use as e.g. documentation examples ("see reference here")
+ Perhaps more maintainable (less complexity?). Code duplication could be tackled via shared structures if necessary.

Leaning a bit towards "one story per control", mostly because it feels like a logical structure - very explicit about what purpose each example serves. Either way, your plan (first functionality with some examples, then nicer examples) sounds reasonable.

Reply:
I agree with you, I think the one per control approach is appealing. I like the idea of it being like documentation. I think we should go down this route.

Additionally I think there is a lot that can be done with the example and it would be great to keep adding to it and improving it. I think there is already an issue on the project board too.

Reply:
Lets move this example into a ControlTypes/ folder which would contain all the control stories etc